data/input_files/default.lua

```diff
@@ -41,41 +41,40 @@ nonlinear_solid = {
     mu = 0.25,
     K  = 5.0,
 
-    -- loading parameters
-    traction = {
-        x = 0.0,
-        y = 1.0e-3,
-        z = 0.0,
-    },
-
     -- initial conditions
     -- initialize x_cur, boundary condition, deformation, and
     -- incremental nodal displacment grid functions by projecting the
     -- VectorFunctionCoefficient function onto them
 
     initial_displacement = {
-        vec_coef = function (v)
-            return Vector.new(0, 0, 0)
-        end
+        constant_vector = {
+            x = 0.0,
+            y = 0.0,
+            z = 0.0
+        }
     },
 
     initial_velocity = {
-        vec_coef = function (v)
-            return Vector.new(0, 0, 0)
-        end
+        constant_vector = {
+            x = 0.0,
+            y = 0.0,
+            z = 0.0
+        }
     },
 
     -- boundary condition parameters
     boundary_conds = {
         ['displacement'] = {
             attrs = {1},
-            vec_coef = function (v)
-                return Vector.new(0, 0, 0)
-            end
+            constant_vector = {
```

Comment:
I like that you removed the "coef" layer.  I assume that was an attempt to debug?

Reply:
Yep. That was verifying that it wasn't getting saved in sidre.